components/collection/drop/MintStepper.vue

```diff
@@ -4,18 +4,22 @@
     v-model="amountToMint"
     data-testid="drop-stepper-container"
     :max="max"
+    :disabled="!isLogIn"
     class="[&>.neo-input]:h-full [&>.neo-input>input]:h-full md:w-[200px]" />
 </template>
 <script lang="ts" setup>
 import { NeoStepper } from '@kodadot1/brick'
 import useHolderOfCollection from '@/composables/drop/useHolderOfCollection'
 
+const { isLogIn } = useAuth()
 const { availableNfts } = useHolderOfCollection()
-const { amountToMint, drop } = storeToRefs(useDropStore())
+const { amountToMint, drop, mintsCount } = storeToRefs(useDropStore())
 
 const show = computed(() => drop.value.type !== 'free')
 
 const max = computed(() =>
-  drop.value.type === 'holder' ? availableNfts.serialNumbers.length : undefined,
+  drop.value.type === 'holder'
+    ? availableNfts.serialNumbers.length
+    : drop.value.max && drop.value.max - mintsCount.value,
 )
```

Comment:
what is this?

Reply:
It sets a max value on the button to prevent a user from entering values like 9999999. It also helps in regulating the size of the button